packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -488,7 +496,27 @@
   }
 
   .is-calendar-focused {
-    outline: var(--terra-date-picker-month-focus-outline, 2px dashed #000);
-    outline-offset: var(--terra-date-picker-month-focus-outline-offset, 0);
```

Comment:
The removal/renaming of this theme variable should be documented in the changelog

Reply:
Added 64aef1f